debian/postinst

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -eu
+
+g_s_helper_user=g-s-helper
```

Comment:
Why is the user called `g-s-helper`? Since this is entirely flatpak specific, that seems wrong.